src/solvers/README.md

```diff
@@ -319,6 +316,8 @@ allocates a new string before calling a primitive.
   * `cprover_string_intern` : Never tested.
   * `cprover_string_is_empty` :
     Should use `cprover_string_length(s) == 0` instead.
+  * `cprover_string_is_suffix` : Should use `cprover_string_is_prefix` with an
+   offset argument.
```

Comment:
Can't this just be removed directly instead of deprecating it? Are there actually any users out there?

Reply:
It is still used when converting from String.endsWith, so there is a little bit more work to do.
We should have a model of endsWith which just calls startsWith then we can remove the preprocessing for endsWith and is_suffix will become dead code.

Reply:
It would be a couple of PRs on other repositories, so it's not difficult changes but they take some time because of CI and such.

Reply:
Ok, thanks for clarifying!